common/src/main/java/org/opensearch/ml/common/output/execute/metrics_correlation/MetricsCorrelationOutput.java

```diff
@@ -66,13 +66,15 @@ public void writeTo(StreamOutput out) throws IOException {
 
     @SneakyThrows
     public XContentBuilder toXContent(XContentBuilder builder, ToXContent.Params params) throws IOException {
+        builder.startObject();
         if (modelOutput != null && modelOutput.size() > 0) {
```

Comment:
if the model doesn't have output, then it won't have the array of inference_result. 
is it better to have an empty inference_result when there is null model output?

IMO, the check of `if (modelOutput != null && modelOutput.size() > 0)` can go after the `startArray` .